.github/workflows/build_test_cross.md

```diff
@@ -0,0 +1,44 @@
+=== Running in docker ===
+
```

Comment:
Maybe add one or two sentences to give some context on what this snippet is doing, where you'll find the resulting binaries, etc.